Makefile

```
@@ -26,24 +26,24 @@ refresh-api:
 	$(DOCKER_RUN) refresh_api.sh
 
 seed-fresh:
-	$(DOCKER_EXEC) "php artisan migrate:fresh --seed"
+	$(DOCKER_API) "php artisan migrate:fresh --seed"
 
 migrate:
-	$(DOCKER_EXEC) "php artisan migrate"
+	$(DOCKER_API) "php artisan migrate"
 
 artisan:
-	$(DOCKER_EXEC) "php artisan $(CMD)"
+	$(DOCKER_API) "php artisan $(CMD)"
 
 watch:
 	$(DOCKER_PNPM) run watch
 
 lint:
-	$(DOCKER_EXEC) "php ./vendor/bin/pint"
+	$(DOCKER_API) "php ./vendor/bin/pint"
 	$(DOCKER_PNPM) lint
 	docker-compose run -w /var/www/html --rm maintenance pnpm run watch
 
 lint-php:
-	$(DOCKER_EXEC) "vendor/bin/pint --test"
+	$(DOCKER_API) "vendor/bin/pint --test"
 
 queue-work:
-	$(DOCKER_EXEC) "runuser -u www-data -- php /home/site/wwwroot/api/artisan queue:work"
+	$(DOCKER_API) "runuser -u www-data -- php /home/site/wwwroot/api/artisan queue:work"
```

Oh, this one is a bit different.  The queue worker has to actually run in the webserver.
```suggestion
	docker-compose exec sh -c "runuser -u www-data -- php /home/site/wwwroot/api/artisan queue:work"
```